src/scss/components/_menus.scss

```diff
@@ -110,7 +110,6 @@ $md-menu-cascading-list-padding: 16px;
     left: 0;
     top: 100%;
     transform-origin: 50% 0;
-    width: 100%;
```

Comment:
i don't know if it should be `min-width: 100%` (i don't know what the implication is) but `SelectField`s do still work fine. 

![optimised](https://cloud.githubusercontent.com/assets/5657668/26685273/9c6659d0-46b7-11e7-8879-98b454b80e79.gif)

(pardon the gif compression)